spec/sidekiq-instrument/worker_spec.rb

```diff
@@ -117,4 +122,109 @@
       ]
     end
   end
+
+  describe 'client & server middleware' do
+    before(:each) do
+      Redis.new.flushall
+      Sidekiq.configure_client do |c|
+        c.client_middleware do |chain|
+          chain.add Sidekiq::Instrument::ClientMiddleware
+        end
+      end
+    end
+
+    after(:each) do
+      Sidekiq.configure_client do |c|
+        c.client_middleware do |chain|
+          chain.remove Sidekiq::Instrument::ClientMiddleware
+        end
+      end
+    end
+
+    context 'successful increment' do
+      let(:expected_dog_options) { { tags: ['queue:default', 'worker:my_worker'] } }
+
+      before do
+        Sidekiq.server_middleware do |chain|
+          chain.add Sidekiq::Instrument::ServerMiddleware
+        end
+      end
+
+      after do
+        Sidekiq.server_middleware do |chain|
+          chain.remove Sidekiq::Instrument::ServerMiddleware
+        end
+      end
+
+      it 'increments the in queue counter' do
+        Sidekiq::Instrument::WorkerMetrics.enabled = true
+        redis = Redis.new
+        expect(redis.hget(worker_metric_name, 'my_worker')).to be nil
+        MyWorker.perform_async
+        expect(redis.hget(worker_metric_name, 'my_worker')).to eq('1')
+        MyWorker.perform_async
+        expect(redis.hget(worker_metric_name, 'my_worker')).to eq('2')
+      end
+
+      it 'increments the DogStatsD failure counter' do
+        expect(
+          Sidekiq::Instrument::Statter.dogstatsd
+          ).to receive(:increment).with('sidekiq.enqueue', expected_dog_options).once
```

Comment:
Nit (Rubocop flags this, too): bump the paren back one level to match its expect( indentation
```suggestion
        ).to receive(:increment).with('sidekiq.enqueue', expected_dog_options).once
```